src/shared/components/boardCard/boardCard.js

```diff
@@ -16,6 +16,7 @@ const BoardCard = ({
     <hr className={styles.hr} />
     <span className={styles.item}>
       <span className={styles.upper}>Role: </span> {role}
+      <p />
```

Comment:
Hmmmmm, can you instead populate this paragraph with a `description` prop and conditionally render it? Let me know if you need some code, it should be a quick implementation.

Reply:
Yes I will make the suggested changes and resubmit. Thanks!